server/controllers/dashboardController.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+import moment from "moment";
+import * as util from "util";
+import models from "../models";
+import { Op } from "sequelize";
+import paginationMeta from "../helpers/paginationHelper";
+import {paginationResponse} from "../utils/formatter";
+import {validFormat} from '../helpers/dateHelpers';
+
+
+const automation = models.Automation;
+/**
+ * Returns dateFunction
+ * @param {object} date - date string
+ * @returns {object} - date object
+ */
+const dateFunction = (date = { endDate: moment(), startDate: moment().subtract(30, 'days')}) => {  
```

Comment:
Since date would always be passed as argument to this function, It might make sense to do Instead:
```
const dateFunction = ({ endDate=moment(), startDate=moment().subtract(30, 'days) })=> {
     **************
     **************
}
```
That way, if date argument doesn't have `endDate` or `startDate`, we can fall back to the default value since the default values specified would not be of help, if we pass an empty object as argument to this function currently

Reply:
Fixed